language/Pattern-Matching.md

```diff
@@ -33,6 +33,7 @@ The following pattern types are supported:
 - Record patterns
 - Named patterns
 - Guards
+- Pattern guards
```

Comment:
Yes, of course I agree that they are intimately related, and it's worth having discussion of them in this file; I just think it's incorrect to include them in this list.